tests/test_url_builder.py

```diff
@@ -278,3 +278,18 @@ def test_shard_strategy_invalid():
 
     # Should not throw an exception
     assert builder.create_url('/users/1.png') is not None
+
+
+def test_create_url_with_sign_with_library_version():
+    with warnings.catch_warnings(record=True) as w:
+        imgix.UrlBuilder('assets.imgix.net', sign_with_library_version=True)
+        assert len(w) == 1
+        assert issubclass(w[-1].category, DeprecationWarning)
+        assert "deprecated" in str(w[-1].message)
+
+
+def test_create_url_with_include_library_param():
```

Comment:
I'd love to see a few more tests here to ensure that `include_library_param=True` works, and to test these two conditions for `sign_with_library_version` as well.